storage/innobase/handler/ha_innodb.cc

```diff
@@ -19977,9 +19979,11 @@ ib_logf(
 		break;
 	case IB_LOG_LEVEL_ERROR:
 		sql_print_error("InnoDB: %s", str);
+		sd_notifyf(0, "STATUS=InnoDB: Error: %s", str);
 		break;
```

Comment:
its informational only to a systemd service list. I think I'm ok with letting the user know an error occured on some connection and they can investigate the error log more. http://man7.org/linux/man-pages/man3/sd_notify.3.html